config/common/traces-limits-v4.4.toml

```diff
@@ -0,0 +1,77 @@
+[traces-limits]
+# Module limits of the beta-v4.4 release (Osaka)
+#
+# Arithmetization module limits
+#
+ADD                 = 262144
+BIN                 = 262144
+BLAKE_MODEXP_DATA   = 16384
+BLOCK_DATA          = 4096
+BLOCK_HASH          = 2048
+BLS_DATA            = 65536
+EC_DATA             = 65536
+EUC                 = 65536
+EXP                 = 65536
+EXT                 = 524288
+GAS                 = 65536
+HUB                 = 2097152
+LOG_DATA            = 65536
+LOG_INFO            = 4096
+MMIO                = 2097152
+MMU                 = 1048576
+MOD                 = 131072
+MUL                 = 65536
+MXP                 = 524288
+OOB                 = 262144
+RLP_ADDR            = 4096
+RLP_TXN             = 131072
+RLP_TXN_RCPT        = 65536
+RLP_UTILS           = 131072
+ROM                 = 8388608
+ROM_LEX             = 1024
+SHAKIRA_DATA        = 65536
+SHF                 = 262144
+STP                 = 16384
+TRM                 = 32768
+TXN_DATA            = 8192
+WCP                 = 262144
+#
+# Precompiles limits
+#
+PRECOMPILE_ECRECOVER_EFFECTIVE_CALLS        = 128
+PRECOMPILE_SHA2_BLOCKS                      = 200
+PRECOMPILE_RIPEMD_BLOCKS                    = 0
+PRECOMPILE_MODEXP_EFFECTIVE_CALLS           = 32
+PRECOMPILE_LARGE_MODEXP_EFFECTIVE_CALLS     = 1
```

Comment:
I don't see 
`PRECOMPILE_MODEXP_EFFECTIVE_CALLS_4096` anymore. 
Can I also remove this?
@letypequividelespoubelles

Reply:
There is still different circuits for small and large modexp. Previously the limit for large modexp was 4096 bits and now its 8192 bits. So I think this configuration line should stay.